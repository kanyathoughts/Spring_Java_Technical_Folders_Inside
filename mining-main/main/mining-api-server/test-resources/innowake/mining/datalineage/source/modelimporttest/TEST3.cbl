       IDENTIFICATION DIVISION.
       PROGRAM-ID. FILDSIB3.
       DATA DIVISION.
       WORKING-STORAGE SECTION.

       01  MY-HEX-CONV.
           05 FILLER-1          PIC X(1) VALUE LOW-VALUES.
           05 MY-HEX-CONV-CHAR  PIC X(1).
       01 WS-ONE PIC 9 VALUE 1.
       01 WS-INPUT PIC X.
       01  MY-HEX-CONV-NUM REDEFINES MY-HEX-CONV PIC 9(4) COMP.
       01 WS-OUTPUTA PIC 9(4).
       01 WS-OUTPUTB PIC 9(4).

       PROCEDURE DIVISION.
           COMPUTE WS-OUTPUTA = WS-INPUT * WS-ONE.
           COMPUTE WS-OUTPUTB = MY-HEX-CONV-NUM * FILLER-1 * 2.
       END PROGRAM FILDSIB3.
