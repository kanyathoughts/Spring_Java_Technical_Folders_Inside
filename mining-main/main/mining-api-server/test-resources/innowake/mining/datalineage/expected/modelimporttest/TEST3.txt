DataFlowNode 
 	 [TYPE = FIELD,
	 NAME = FILLER-1,
	 LOCATION = ModuleLocation[offset=151,length=47],
	 MODULEID = 2002,
	 RELATEDFIELDS = [MY-HEX-CONV, MY-HEX-CONV-NUM],
	 READACCESSES = [Compute],
	 WRITEACCESSES = []]

DataFlowNode 
 	 [TYPE = FIELD,
	 NAME = MY-HEX-CONV,
	 LOCATION = ModuleLocation[offset=123,length=16],
	 MODULEID = 2002,
	 RELATEDFIELDS = [FILLER-1, MY-HEX-CONV-CHAR, MY-HEX-CONV-NUM],
	 READACCESSES = [],
	 WRITEACCESSES = []]

DataFlowNode 
 	 [TYPE = FIELD,
	 NAME = MY-HEX-CONV-CHAR,
	 LOCATION = ModuleLocation[offset=210,length=30],
	 MODULEID = 2002,
	 RELATEDFIELDS = [MY-HEX-CONV, MY-HEX-CONV-NUM],
	 READACCESSES = [],
	 WRITEACCESSES = []]

DataFlowNode 
 	 [TYPE = FIELD,
	 NAME = MY-HEX-CONV-NUM,
	 LOCATION = ModuleLocation[offset=306,length=56],
	 MODULEID = 2002,
	 RELATEDFIELDS = [FILLER-1, MY-HEX-CONV, MY-HEX-CONV-CHAR],
	 READACCESSES = [Compute],
	 WRITEACCESSES = []]

DataFlowNode 
 	 [TYPE = FIELD,
	 NAME = WS-INPUT,
	 LOCATION = ModuleLocation[offset=280,length=18],
	 MODULEID = 2002,
	 RELATEDFIELDS = [],
	 READACCESSES = [Compute],
	 WRITEACCESSES = []]

DataFlowNode 
 	 [TYPE = FIELD,
	 NAME = WS-ONE,
	 LOCATION = ModuleLocation[offset=248,length=24],
	 MODULEID = 2002,
	 RELATEDFIELDS = [],
	 READACCESSES = [Compute],
	 WRITEACCESSES = []]

DataFlowNode 
 	 [TYPE = FIELD,
	 NAME = WS-OUTPUTA,
	 LOCATION = ModuleLocation[offset=370,length=23],
	 MODULEID = 2002,
	 RELATEDFIELDS = [],
	 READACCESSES = [],
	 WRITEACCESSES = [Compute]]

DataFlowNode 
 	 [TYPE = FIELD,
	 NAME = WS-OUTPUTB,
	 LOCATION = ModuleLocation[offset=401,length=23],
	 MODULEID = 2002,
	 RELATEDFIELDS = [],
	 READACCESSES = [],
	 WRITEACCESSES = [Compute]]

DataFlowNode 
 	 [TYPE = STATEMENT,
	 NAME = Compute,
	 LOCATION = ModuleLocation[offset=464,length=38],
	 MODULEID = 2002,
	 RELATEDFIELDS = [],
	 READACCESSES = [WS-INPUT, WS-ONE],
	 WRITEACCESSES = [WS-OUTPUTA]]

DataFlowNode 
 	 [TYPE = STATEMENT,
	 NAME = Compute,
	 LOCATION = ModuleLocation[offset=515,length=51],
	 MODULEID = 2002,
	 RELATEDFIELDS = [],
	 READACCESSES = [FILLER-1, MY-HEX-CONV-NUM],
	 WRITEACCESSES = [WS-OUTPUTB]]

